iqb/.htaccess

```
@@ -1,4 +1,7 @@
 RewriteEngine on
+RewriteRule ^v25/(.*) https://skohub.io/iqb-vocabs/v25/heads/master/w3id.org/iqb/v25/$1 [R=302,L]
+RewriteRule ^v26/(.*) https://skohub.io/iqb-vocabs/v26/heads/master/w3id.org/iqb/v26/$1 [R=302,L]
+RewriteRule ^v27/(.*) https://skohub.io/iqb-vocabs/v27/heads/master/w3id.org/iqb/v27/$1 [R=302,L]
```

Consider replacing all these with a single rule using more pattern matching (untested patch):
```suggestion
RewriteRule ^v([0-9]+)/(.*) https://skohub.io/iqb-vocabs/v$1/heads/master/w3id.org/iqb/v$1/$2 [R=302,L]
```